core/src/Authorization.html

```diff
@@ -1,31 +1,74 @@
 
 {#if !isHidden}
-<ul class="fd-menu__list">
-  {#if isLoggedIn}
-  {#if showUserInfo}
-  <li>
-    <a
-      aria-label="Username"
-      class="fd-menu__item"
-    >{userInfo.name ? userInfo.name : userInfo.email}</a>
-  </li>
-  {/if}
-  <li on:click="onLogoutClick()">
-    <a aria-label="Logout" class="fd-menu__item">Logout</a>
-  </li>
-  {:else}
-  <li on:click="startAuthorization()">
-    <a aria-label="Login" class="fd-menu__item">Login</a>
-  </li>
-  {/if}
-</ul>
+<nav class="fd-menu">
+  <ul class="fd-menu__list">
+    {#if isLoggedIn}
+    {#if showUserInfo}
+    <li>
+      <a
+        aria-label="Username"
+        class="fd-menu__item"
```

Comment:
I think username should be somehow distinguished from the list (not be a list item), as shown on the ticket's screenshot.

Reply:
I will keep it in the list and use a bigger font size so that it is more prominent than the other elements

Reply:
Ok but keep on mind that the user name can be loooooonger.

Reply:
Long names would just make the dropdown wider.